sloop-helm-charts/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
0.2

Reply:
actually, what is appVersion vs version?

Reply:
appVersion: The version of the app that this contains (optional). This needn't be SemVer.
https://github.com/helm/helm/blob/master/docs/charts.md#the-appversion-field